libs/libarchopenfpga/src/bitstream_setting.cpp

```diff
@@ -178,6 +182,12 @@ void BitstreamSetting::add_non_fabric_pb(const std::string& pb,
   }
 }
 
+void BitstreamSetting::add_path_bit_setting(const std::string& path,
+                                            const bool value) {
```

Comment:
Use ``const bool&`` to avoid memory copy

Reply:
Done via https://github.com/chungshien/OpenFPGA/commit/2a3d69aded840c54212dd9652670ab54e9ab1222